gateway-ha/src/test/java/io/trino/gateway/ha/router/TestExternalUrlQueryHistoryMySql.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.gateway.ha.router;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.MySQLContainer;
+
+public class TestExternalUrlQueryHistoryMySql
+        extends BaseExternalUrlQueryHistoryTest
+{
+    public TestExternalUrlQueryHistoryMySql()
+    {
+        super(createMySqlContainer());
+    }
+
+    private static JdbcDatabaseContainer<?> createMySqlContainer()
```

Comment:
Remove redundant `createMySqlContainer`, `createOracleContainer`, `createPostgresqlContainer` methods. We can start the container in `BaseExternalUrlQueryHistoryTest`.

Reply:
I pushed a new commit. Please take a look at that.